.gitignore

```diff
@@ -1,3 +1,4 @@
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
+.git/
```

Comment:
This should not be required. Git normally ignores `.git` just fine?

Reply:
Oops, meant to put it in the .dockerignore :)